src/Resources/contao/library/Contao/Search.php

```diff
@@ -349,23 +343,29 @@ public static function searchFor($strKeywords, $blnOrSearch=false, $arrPid=array
 				case '"':
 					if ($strKeyword = trim(substr($strKeyword, 1, -1)))
 					{
-						$arrPhrases[] = '[[:<:]]' . str_replace(array(' ', '*'), array('[^[:alnum:]]+', ''), $strKeyword) . '[[:>:]]';
+						$arrPhrases[] = str_replace(' ', '[^[:alnum:]]+', preg_quote($strKeyword));
```

Comment:
Do we no longer need the word boundaries?

Reply:
They don’t work for many cases like `"Google+"` or `"使用"`.
Also `"oogle+"` would still not match because the query first searches for `oogle` in `tl_search_index` before it does the regex search.

Reply:
Ok. 👍